packages/react-router/modules/index.js

```diff
@@ -7,3 +7,4 @@ export StaticRouter from './StaticRouter'
 export Switch from './Switch'
 export matchPath from './matchPath'
 export withRouter from './withRouter'
+export { resolveLocation } from './resolve'
```

Comment:
Ideally a user would never have to use `resolvePath` because the `<Route>` component does it for you. It is really just a less powerful version of `resolveLocation` that just joins two strings if the `path` doesn't begin with a slash.

Because this PR doesn't actually add the automatic resolving for a `<Route>`, it would make sense to export `resolvePath`, but I'm not sure how valuable it would be once support for automatic resolving is added.